src/filesim.rs

```diff
@@ -185,11 +199,19 @@ fn main_impl() -> Result<()> {
                 }
             }
         }
+        k.kbd_out
+            .log
+            .end(config_sim_file, args.sim_appendix.clone());
     }
 
     Ok(())
 }
 
+#[cfg(not(feature = "simulated_output"))]
+fn main() -> Result<()> {
```

Comment:
I suppose this can be left to later, e.g. when proper support is builtin for reading from stdin for piping from programs.

I don't think the binary needs to do nothing in the case that simulated output is not enabled. If this file is just going to do nothing though, perhaps it should instead be a compile error: https://doc.rust-lang.org/std/macro.compile_error.html . Though if that makes CI fail, I think it's fine to leave it as-is.

Reply:
Yeah, to me this binary should simply not exist without the simulated feature, but don't know to make it work with these cfg conditionals.

Don't understand the benefit of the macro since it just always makes the compilation fail even locally since it does not prevent cargo from trying to compile the `kanata_filesim` binary if there is no `simulated_output` feature?